### PR TITLE
chore(quick-inventory): send quick inventory to output bucket

### LINKED ISSUE #2333: [Bug]: using -D to send output to S3 does not work with -i for inventory
### Steps to Reproduce

1 - prowler -i -R rolename -D bucketname
2 - aws
3 - all work but nothing is pushed to s3 while if we remove -i we got the security check in S3

### Expected behavior

same as we are getting output files when doing security check we get inventory 

### Actual Result with Screenshots or Logs

no error in logs

### How did you install Prowler?

From pip package (pip install prowler)

### Environment Resource

ec2 instance

### OS used

ubuntu

### Prowler version

3.3.4

### Pip version

pip 22.3.1 from /usr/local/lib/python3.9/site-packages/pip (python 3.9)

### Context

_No response_

### PR DESCRIPTION
**Pull request by MrCloudSec**
### Description

Send quick inventory to output bucket

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prowler/providers/aws/lib/quick_inventory/quick_inventory.py [hunk 1]
```diff
@@ -244,10 +247,14 @@ def create_output(resources: list, audit_info: AWS_Audit_Info, output_directory:
...
-    with open(output_file + json_file_suffix, "w") as outfile:
+    with open(
+        args.output_directory + "/" + output_file + json_file_suffix, "w"
+    ) as outfile:
         outfile.write(json_object)
 
-    csv_file = open(output_file + csv_file_suffix, "w", newline="")
+    csv_file = open(
+        args.output_directory + "/" + output_file + csv_file_suffix, "w", newline=""
+    )
```
Comment (jfagoagas) [inline prowler/providers/aws/lib/quick_inventory/quick_inventory.py:257]: Why the `json` is using the context manager and the `csv` don't?

File: prowler/providers/aws/lib/quick_inventory/quick_inventory.py [hunk 2]
```diff
@@ -261,8 +268,25 @@ def create_output(resources: list, audit_info: AWS_Audit_Info, output_directory:
...
+
+    # Send output to S3 if needed (-B / -D)
+    for mode in ["json", "csv"]:
+        if args.output_bucket or args.output_bucket_no_assume:
+            output_bucket = args.output_bucket
+            bucket_session = audit_info.audit_session
+            # Check if -D was input
+            if args.output_bucket_no_assume:
+                output_bucket = args.output_bucket_no_assume
+                bucket_session = audit_info.original_session
```
Comment (jfagoagas) [inline prowler/providers/aws/lib/quick_inventory/quick_inventory.py]:
```suggestion
        if args.output_bucket or args.output_bucket_no_assume:
            # Check if -B was input
            if args.output_bucket:
                output_bucket = args.output_bucket
                bucket_session = audit_info.audit_session
            # Check if -D was input
            elif args.output_bucket_no_assume:
                output_bucket = args.output_bucket_no_assume
                bucket_session = audit_info.original_session
```

**Review by jfagoagas — CHANGES_REQUESTED** [review]

File: prowler/providers/aws/lib/quick_inventory/quick_inventory.py (hunk 1 above)
Comment (MrCloudSec) [reply prowler/providers/aws/lib/quick_inventory/quick_inventory.py:257]: It was already like that

**Review by jfagoagas — APPROVED** [review]
🚀

**Pull request merged**